packages/utils/package.json

```diff
@@ -15,13 +15,17 @@
     "test": "jest --passWithNoTests"
   },
   "dependencies": {
+    "@ceramicnetwork/http-client": "^2.15.0",
```

Comment:
why the new dependencies?

Reply:
A library conflict where it says, "Can't assign CeramiApi to CeramicApi, their signatures are different." So, trying to get everything using the same dependencies.